source/math/FloatRandomGenerator.ooc

```diff
@@ -29,24 +30,21 @@ FloatRandomGenerator: abstract class {
 FloatUniformRandomGenerator: class extends FloatRandomGenerator {
 	_state: UInt
 	_min, _max, _rangeCoefficient: Float
+	_unsignedIntMaxAsFloat : static Float = 4294967295U as Float
```

Comment:
Actually, I don't see this being used more than once. And it has nothing to do with the class, really, so just put type `UINT_MAX as Float` instead of `This _unsignedIntMaxAsFloat` below.